crypto-square/example.lua

```diff
@@ -0,0 +1,51 @@
+local function normalized_plaintext(plaintext)
+  return plaintext:lower():gsub('%p*%s*', '')
+end
+
+local function size(plaintext)
+  local normalized_length = #normalized_plaintext(plaintext)
+
+  for i = 1, math.huge do
+    if i ^ 2 >= normalized_length then return i end
+  end
+end
```

Comment:
I am not following, I thought this was simply `math.ceil(math.sqrt(#normalized_plaintext(plaintext)))`.